Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -312,6 +312,8 @@ struct PerWebProcessState {
 
     NSUInteger _focusPreservationCount;
     NSUInteger _activeFocusedStateRetainCount;
+
+    RetainPtr<NSArray<NSNumber *>> _originalAllowedTouchTypes;
```

Comment:
Nit: Maybe call this `_scrollViewDefaultAllowedTouchTypes` to make it clear it's about the scroll view?